### PR TITLE
Loops (for, for in, while, do)

### PR DESCRIPTION
**Pull request by mprobst**

### CODE REVIEW

File: test/DartTest.ts [hunk 1]
```diff
@@ -113,6 +113,19 @@ describe('transpile to dart', function() {
       expectTranslate('switch(x) { case 1: break; case 2: break; default: break; }')
           .to.equal(' switch ( x ) { case 1 : break ; case 2 : break ; default : break ; }');
     });
+
+    it('tranlates for loops', function() {
```
Comment (alexeagle) [inline test/DartTest.ts]: tranlates -> translates (throughout)

**Comment by alexeagle** [comment]
Note I didn't merge this one because github says it needs a manual merge

File: test/DartTest.ts (hunk 1 above)
Comment (mprobst) [inline test/DartTest.ts]: Fixed.

**Comment by mprobst** [comment]
Fixed comments, rebased, now merging.

**Pull request merged**